### PR TITLE
Updated Config and English language files

### PR DESCRIPTION
**Pull request by rocco27**
- Support for Conan Exiles game server
- Corrigations for English language
- Polished Rust game server
- RCON fix for Call of Duty 4 game server
- Max. player limit increased in Squad game server
- LGSL fixed for ARK: Survival Evolved (new xml comes asap)

### CODE REVIEW

File: lang/English/modules/server.php [hunk 1]
```diff
@@ -1,137 +1,137 @@
...
+define('edit_range', "Edit range");
+define('delete_range', "Delete range");
+define('home_id', "Home ID");
+define('home_path', "Home path");
+define('game_type', "Game type");
+define('port', "Port");
+define('invalid_values', "Invalid values.");
+define('ports_in_range_already_arranged', "Ports in range already arranged.");
+define('ports_range_already_configured_for', "Ports range already configured for %s.");
+define('ports_range_added_successfull_for', "Ports range added successful for %s.");
```
Comment (Zorrototo) [inline lang/English/modules/server.php]: successfully

File: lang/English/modules/server.php [hunk 2]
```diff
@@ -1,137 +1,137 @@
...
+define('ports_range_deleted_successfull', "Ports range deleted successful.");
```
Comment (Zorrototo) [inline lang/English/modules/server.php]: successfully

File: lang/English/modules/server.php [hunk 3]
```diff
@@ -1,137 +1,137 @@
...
+define('ports_range_edited_successfull_for', "Ports range edited successful for %s.");
```
Comment (Zorrototo) [inline lang/English/modules/server.php]: successfully

Comment (Zorrototo) [inline lang/English/modules/server.php]: successfully

Comment (rocco27) [inline lang/English/modules/server.php]: Correct. I forgot to check the sentences.

**Pull request merged**